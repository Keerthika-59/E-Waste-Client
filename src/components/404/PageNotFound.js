import React from 'react';
import {Image, Button} from 'react-bootstrap';

import {Link} from 'react-router-dom';

const PageNotFound = () => {
    return (
        <div className = "text-center">
            <Image
                height = {650}
                width = {650}
                src= 'https://miro.medium.com/max/5120/1*DeBkx8vjbumpCO-ZkPE9Cw.png'
            />
            
            <Link
                to="/"
            >
            <Button 
            variant = "info"
            > 
            Back to Home
            </Button>
            </Link>
        </div>
    )
}

export default PageNotFound
