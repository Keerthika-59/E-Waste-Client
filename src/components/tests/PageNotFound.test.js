import React from "react";
import { shallow } from 'enzyme';
import ReactDOM from "react-dom";
import { Route } from "react-router-dom";
import PageNotFound from "../404/PageNotFound";

test("renders page not found", () => {
    const div = document.createElement("div");
    <Route>
      ReactDom.render(
      <PageNotFound />
      ,div);
    </Route>;
    ReactDOM.unmountComponentAtNode(div);
  });
  test('should test PageNotFound component', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
   });