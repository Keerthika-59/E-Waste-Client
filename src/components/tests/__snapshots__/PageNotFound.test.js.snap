// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test PageNotFound component 1`] = `
<div
  className="text-center"
>
  <Image
    fluid={false}
    height={650}
    rounded={false}
    roundedCircle={false}
    src="https://miro.medium.com/max/5120/1*DeBkx8vjbumpCO-ZkPE9Cw.png"
    thumbnail={false}
    width={650}
  />
  <Link
    to="/"
  >
    <Button
      active={false}
      disabled={false}
      variant="info"
    >
      Back to Home
    </Button>
  </Link>
</div>
`;
